src/lib/coda_base/receipt_chain_database.ml

```diff
@@ -7,10 +7,6 @@ module Payment = struct
   module T = struct
     include User_command
 
-    type payload = Payload.Stable.V1.t
-
-    let payload {payload; _} = payload
-
     let compare this that = compare this.payload that.payload
```

Comment:
Sorry! Can you fix this too? Define compare on the User_command and use the User_command directly instead of referring to it as payment (just makes the code easy to read when we look back on this)